widgetssdk/src/main/java/com/glia/widgets/core/configuration/GliaSdkConfigurationManager.java

```diff
@@ -21,7 +19,8 @@ public class GliaSdkConfigurationManager {
     private ScreenSharing.Mode screenSharingMode = null;
     private String companyName = null;
     private String legacyCompanyName = null;
-    private boolean useOverlay = true;
+    private boolean enableBubbleOutsideApp = true; // default values
```

Comment:
Shouldn't this be false because this requires permission? Default true in app only?

Reply:
No, I think that both new parameters should be `true` by default because `useOverlay` was `true` by default.
And `useOverlay` is now split into `enableBubbleOutsideApp` and `enableBubbleInsideApp`, so if we are to keep the same logic, both should inherit `true` from `useOverlay`.